Text Extraction/Extract-each-characters-from-an-existing-PDF-document/.NET/Extract-each-characters-from-an-existing-PDF-document/Program.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Syncfusion.Drawing;
+using Syncfusion.Pdf;
```

Comment:
The title is not proper. Can we add like Get-text-glyph-details-from-extract-text